### PR TITLE
[REVIEW] Open cypher integrate parser

### PR DESCRIPTION
**Pull request by jwyles**
Integrated the libcypher-parser conda package into the project.

### CODE REVIEW

**Comment by afender** [comment]
Not passing cmake yet 
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.

Please set them or make sure they are set and tested correctly in the CMake files:

LIBCYPHERPARSER_LIBRARY
```

File: conda/environments/cugraph_dev_cuda10.yml [hunk 1]
```diff
@@ -3,8 +3,10 @@ channels:
 - rapidsai/label/cuda10.0
 - nvidia/label/cuda10.0
 - rapidsai-nightly/label/cuda10.0
+- rapidsai/label/thirdparty
 - conda-forge
 dependencies:
+- libcypher-parser
```
Comment (afender) [inline conda/environments/cugraph_dev_cuda10.yml]: `libcypher-parser` should also be in 10.1 dev env file once it is merged.

You probably need to update `build.sh` under ci and `meta.yml` under conda.

**Comment by rlratzel** [comment]
rerun tests

**Comment by rlratzel** [comment]
rerun tests

**Pull request merged**